tla/ccfraft.tla

```diff
@@ -729,6 +735,11 @@ AdvanceCommitIndex(i) ==
               \* Retire if i is not in active configuration anymore
               /\ IF i \notin configurations[i][Min(DOMAIN new_configurations)]
                  THEN /\ state' = [state EXCEPT ![i] = RetiredLeader]
+                      /\ LET msg == [type          |-> ProposeVoteRequest,
+                                     term          |-> currentTerm[i],
+                                     source        |-> i,
+                                     dest          |-> i]
+                         IN Send(msg)
```

Comment:
How about something like:

```tla
i = CHOOSE n \in { n \in Servers : \A m \in Servers : matchIndex[i][n] >= matchIndex[i][m] }: n \in MaxConfigurationIndex(i) => Send(msg)
```

`n \in MaxConfigurationIndex(i)` might not exactly capture what `raft.h` says about tie breaking.

Reply:
Fixed with help from @heidihoward, thank you!